src/Repository/RepositoryInterface.php

```diff
@@ -14,6 +14,15 @@
     public function find($id);
 
     /**
+     * Find multiple objects by identifier
+     *
+     * @param array $ids
+     *
+     * @return array
+     */
+    public function findMultiple(array $ids);
```

Comment:
Actually... hm... should it maybe be `findByIds` to make the name indicative of the parameter type?